.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,17 @@
+on:
+  schedule:
+    - cron:  '0 2 * * *'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm install
+      - run: npm run compile
+      - run: echo $KEYFILE_CONTENTS | base64 --decode > keyfile.json
```

Comment:
I would be tempted to make this explicitly, `> /tmp/keyfile.json`.

Reply:
let's give it a shot